### PR TITLE
Project Submission Pull Request

### PR DESCRIPTION
**Pull request by etridgely**

### CODE REVIEW

**Comment by etridgely** [comment]
Finished

File: challenges/classes.js [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+    surfaceArea() {
+        return (2 * (this.length * this.width + this.length * this.height + this.width * this.height));
+    }
+  }
+  
+class CubeMaker extends CuboidMaker {
+    constructor(attributes) {
+        super(attributes);
+    }
+}
```
Comment (thereactgirl) [inline challenges/classes.js:21]: Awesome job on the stretch. :) I guess you didn't need any new methods. :)

File: challenges/functions.js [hunk 2]
```diff
@@ -7,18 +7,32 @@
...
+  console.log(x + y);
+}
+
+const multiply = function(x,y) {
+  console.log(x * y);
+}
+
+const greeting = function(firstName, lastName) {
+  console.log(`Hello ${firstName} ${lastName}, nice to meet you!`);
+}
```
Comment (thereactgirl) [inline challenges/functions.js:30]: Great job on the callbacks! Everything is console logging correctly!

File: challenges/objects-arrays.js [hunk 3]
```diff
@@ -49,7 +74,10 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
+const universities = graduates.map((school) => {
+  return school.university;
+});
+universities.sort();
 console.log(universities)
```
Comment (thereactgirl) [inline challenges/objects-arrays.js:82]: Perfect map method..

File: challenges/objects-arrays.js [hunk 4]
```diff
@@ -58,12 +86,16 @@ The resulting contact information should have a space between the first name and
...
+});
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = graduates.filter((u) => {
+  return u.university.includes("Uni");
+});
 console.log(uni);
```
Comment (thereactgirl) [inline challenges/objects-arrays.js:100]: You got these advanced array methods down!

**Review by thereactgirl — COMMENTED** [review]
You met all the objectives and stretch goals!

**Pull request merged**